opentech/apply/projects/templates/application_projects/project_simplified_detail.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base-apply.html" %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
+        <h5 class="heading heading--meta">
+            <span>{{ object.submission.page }}</span>
+            <span>{{ object.submission.round }}</span>
+            <span>Lead: {{ object.lead }}</span>
+        </h5>
+    </div>
+</div>
+
+<div class="wrapper wrapper--large">
+
+    {% block notifications %}
+    {% endblock %}
+    <div class="wrapper wrapper--sidebar">
+        <article class="wrapper--sidebar--inner">
+            <h3>Project Information</h3>
+            <div class="grid grid--proposal-info">
+                <div>
+                    <h5>Proposed start date</h5>
+                    <p>{{ object.proposed_start|date:"j F Y"|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Project Proposed end date</h5>
+                    <p>{{ object.proposed_end|date:"j F Y"|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Legal name</h5>
+                    <p>{{ object.contact_legal_name|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Email</h5>
+                    {% if object.contact_email %}
+                    <a href="mailto:{{ object.contact_email }}">{{ object.contact_email }}</a>
```

Comment:
Mailto doesn't make sense here i don't think. page is designed for printing not interaction.

Reply:
Fixed and rebased.